go.mod

```diff
@@ -2,9 +2,15 @@ module github.com/siddontang/go-mysql
 
 require (
 	github.com/BurntSushi/toml v0.3.1
+	github.com/go-sql-driver/mysql v1.4.1
```

Comment:
em, why do you introduce these dependencies, seem they are not relevant here.

Reply:
these dependencies are introduced after running `go mod tidy`.

for example:

```
go mod why -m github.com/go-sql-driver/mysql
# github.com/go-sql-driver/mysql
github.com/siddontang/go-mysql/server
github.com/siddontang/go-mysql/server.test
github.com/go-sql-driver/mysql
```

`github.com/go-sql-driver/mysql` is introduce at https://github.com/siddontang/go-mysql/blob/7263f5610b7a05bfb0ceff83e99ada9f5287a904/server/server_test.go#L13